packages/syft/src/syft/service/worker/worker_image_service.py

```diff
@@ -39,15 +39,15 @@ def __init__(self, store: DocumentStore) -> None:
         self.stash = SyftWorkerImageStash(store=store)
 
     @service_method(
-        path="worker_image.submit_dockerfile",
-        name="submit_dockerfile",
+        path="worker_image.submit_container_image",
+        name="submit_container_image",
         roles=DATA_OWNER_ROLE_LEVEL,
```

Comment:
Have we reviewed these API changes by Irina?

Reply:
Yes alr gone over this with Shubham and gonna bring this up to Irina before merging.